diff -u -r glfw-3.2.1-a/CMakeLists.txt glfw-3.2.1-b/CMakeLists.txt
--- glfw-3.2.1-a/CMakeLists.txt	2018-08-15 18:56:07.200726589 -0600
+++ glfw-3.2.1-b/CMakeLists.txt	2018-08-15 18:56:17.824331306 -0600
@@ -65,7 +65,8 @@
 list(APPEND CMAKE_MODULE_PATH "${GLFW_SOURCE_DIR}/CMake/modules")
 
 find_package(Threads REQUIRED)
-find_package(Vulkan)
+# Patched out!
+#find_package(Vulkan)
 
 if (GLFW_BUILD_DOCS)
     set(DOXYGEN_SKIP_DOT TRUE)
